### DISCUSSION TITLE
DNS Queries for "." every second?

### DISCUSSION THREAD

**Discussion by tknorris** [AdguardTeam/AdGuardHome#6152] [Q&A] [answered]
I recently migrated to Adguard Home on linux/docker from Adguard Home in a FreeBSD Jail on FreeNAS.

In the new install I get a query for the "." domain about once every second. I never saw that on my old install. Anyone have any idea WHY that might be happening? _MOST_ of the time it's from the docker containers own docker network, but it also occasionally comes from my router. If it's needed, is there a way to stop it from appearing in the logs? From googling, I understand this is a ROOT query, but I don't understand why I've never seen it before. Is it triggered by running the docker container in it's own network?

![Cursor_and_AdGuard_Home](https://github.com/AdguardTeam/AdGuardHome/assets/1934551/2d8701d9-7aac-454f-836b-84698c5c5424)

**Comment by ainar-g** [comment]
See #5995.  Some clients seem to send them, and you can either block them completely or remove them from the logs and statistics.

**Reply by tknorris** [reply, marked as answer]
Thanks, that's exactly what I needed. Issue #5990 had the info I needed to filter it out.
